src/test/java/net/sf/jsqlparser/statement/select/SpeedTest.java

```diff
@@ -23,7 +23,10 @@
 import net.sf.jsqlparser.test.TestException;
 import net.sf.jsqlparser.util.TablesNamesFinder;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
 
+@Execution(ExecutionMode.CONCURRENT)
```

Comment:
This parallel execution would make the numbers from this speed test meaningless.

Reply:
Am not entirely clear, how and why. But I have removed that annotation from the Speed Test.